neo/SmartContract/InteropService.cs

```diff
@@ -18,6 +18,9 @@ namespace Neo.SmartContract
 {
     public static partial class InteropService
     {
+        public const int MaxStorageKeySize = 512;
```

Comment:
byte.MaxValue is not enough?
```suggestion
        public const int MaxStorageKeySize = byte.MaxValue;
```